src/main/java/org/apache/sysml/api/DMLScript.java

```diff
@@ -102,66 +109,58 @@
 		HYBRID_SPARK,   // execute matrix operations in CP or Spark   
 		SPARK			// execute matrix operations in Spark
 	}
+
+	/**
+	 * Set of DMLOptions that can be set through the command line
+	 * and {@link org.apache.sysml.api.mlcontext.MLContext}
+	 * The values have been initialized with the default values
+	 * Despite there being a DML and PyDML, this class is named DMLOptions
+	 * to keep it consistent with {@link DMLScript} and {@link DMLOptions}
+	 */
+	public static class DMLOptions {
+		public Map<String, String>  argVals 			= new HashMap<>();	// Arguments map containing either named arguments or arguments by position for a DML program
+		public String               configFile    = null;							// Path to config file if default config and default config is to be overriden
+		public boolean              clean         = false;						// Whether to clean up all SystemML working directories (FS, DFS)
+		public boolean              stats         = false;						// Whether to record and print the statistics
+		public int                  statsCount    = 10;								// Default statistics count
+		public Explain.ExplainType  explainType  	= Explain.ExplainType.NONE;		// Whether to print the "Explain" and if so, what type
+		public DMLScript.RUNTIME_PLATFORM execMode = OptimizerUtils.getDefaultExecutionMode();	// Execution mode standalone, MR, Spark or a hybrid
+		public boolean              gpu           = false;						// Whether to use the GPU
+		public boolean              forceGPU      = false;						// Whether to ignore memory & estimates and always use the GPU
+		public boolean              debug         = false;						// to go into debug mode to be able to step through a program
+		public ScriptType           scriptType    = ScriptType.DML;		// whether the script is a DML or PyDML script
+		public String               filePath 			= null;							// path to script
+		public String 							script				= null;							// the script itself
+		public boolean              help          = false;						// whether to print the usage option
+
+		public final static DMLOptions defaultOptions = new DMLOptions();
```

Comment:
Can you fix the spacing in this section?  It would be better to just use spaces to align everything.